lms/templates/ccx/schedule.html

```diff
@@ -72,26 +72,28 @@
     <h2>${_('Schedule a Unit')}</h2>
     <form role="form" id="add-unit" name="add-unit" class="ccx-form">
       <div class="field">
-        <b>${_('Section')}</b><br/>
-        <select name="chapter"></select>
+        <label for="chapter"><b>${_('Section')}</b></label>
+        <select name="chapter" id="chapter"></select>
       </div>
       <div class="field">
-        <b>${_('Subsection')}</b><br/>
-        <select name="sequential"></select>
+        <label for="sequential"><b>${_('Subsection')}</b></label>
+        <select name="sequential" id="sequential"></select>
       </div>
       <div class="field">
-        <b>${_('Unit')}</b><br/>
-        <select name="vertical"></select>
+        <label for="vertical"><b>${_('Unit')}</b></label>
+        <select name="vertical" id="vertical"></select>
       </div>
       <div class="field datepair">
-        <b>${_('Start Date')}</b><br/>
-        <input placeholder="Date" type="date" class="date" name="start_date"/>
-        <input placeholder="time" type="time" class="time" name="start_time"/>
-      </div>
+        <label for="start_date"><b>${_('Start Date')}</b></label>
+        <input placeholder="Date" type="date" class="date" name="start_date" id="start_date"/>
+        <label for="start_time" class="sr">${_('Start Time')}</label>
+        <input placeholder="time" type="time" class="time" name="start_time" id="start_time" />
+        </div>
       <div class="field datepair">
-        <b>${_('Due Date')}</b> ${_('(Optional)')}<br/>
-        <input placeholder="Date" type="date" class="date" name="due_date"/>
-        <input placeholder="time" type="time" class="time" name="due_time"/>
+        <label for="due_date"><b>${_('Due Date')}</b> ${_('(Optional)')}</label>
+        <input placeholder="Date" type="date" class="date" name="due_date" id="due_date"/>
+        <label for="due_time" class="sr">${_('Due Time')}</label>
```

Comment:
Added label for due time with **class=sr**, it is only visible to screen readers.